hack/lib/cleanup.sh

```diff
@@ -36,7 +36,6 @@ function os::cleanup::all() {
 	os::cleanup::truncate_large_logs
 
 	if [[ -z "${SKIP_TEARDOWN:-}" ]]; then
-		os::cleanup::containers
```

Comment:
i think there is a lot more that can/should be removed from this cleanup routine given that this is used completely differently from how it used to be, but this is one i'm reasonably confident of and takes a bunch of time to run locally.